kubernetes/db-migrate-production.tmpl

```diff
@@ -0,0 +1,23 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: __JOB_NAME__
+spec:
+  template:
+    spec:
+      containers:
+      - name: tove-db-migrate-staging
```

Comment:
Do we still want this to be 'staging'? eg. tove-db-migrate-staging

Reply:
aw heck, this is why I need reviews. thank you!